[AD-382] Tool for viewing current content of acid-state (#350)
serokell/ariadne: acid-view/README.md

# AcidView 

> Helper utility that prints content of Acid DB to terminal or file

Reviewer: ```suggestion
Helper utility that prints contents of Acid DB to the terminal or a file.
```

> One can view state of the database of n events ago (up to last checkpoint).

Reviewer: ```suggestion
One can view the state of the database `n` events ago (up to the last checkpoint).
```

```
Usage: acid-view [--dbpath DBPath] [-p|--position Steps]
                 [-s|--show TypeOfOutput] [-f|--file FileToSave]
```

> examples: 

Reviewer: ```suggestion
Example usage:
```

```
acidView --dbpath /home/username/DBPath -p 2 -s addr
``` 

>    It will print to console addresses, that was in DB 2 events ago.

Reviewer: ```suggestion
Prints to stdout addresses that were in the DB 2 events ago.
```

```
acidView -f DBDump.txt
``` 

> It will save to file the whole content of DB (from default path).

Reviewer: ```suggestion
Saves full contents of the DB (from default path) to `DBDump.txt`.
```

# AcidView 
Helper utility that prints content of Acid DB to terminal or file 

One can view state of the database of n events ago (up to last checkpoint). 

Usage 
  acidView <path> <pos>(optional) <what to show>(optional) <filename>(optional) 

Path: path with DB. By default it is "~/.local/share/ariadne/.wallet-db/"
pos: Position in history of events (from the head). By default it is 0 (current state)
what to show: What part of DB should be printed ("wal" - wallets, "acc" - accounts,
                 "addr" - addresses, "all" - the whole database). By default it is "all"

> filename: Write it if you want to save info to file. Otherwise content of DB will be printed to console. 

Author: Probably you are right. Moved that part.
